HW5/src/Main/app.ts

```diff
@@ -1,16 +1,17 @@
-import { modal } from "./modal.js";
```

Comment:
It's better to place the entrypoint in the src/ folder, without additional folder. Also, it's better to name it as index.ts - it's just common practice
Also, you didn't implement the component approach from the hw5 (when you have saparated functions that return html elements and you have a state to rerender all the componets), so you cannot divide project to components without the components. You have several utils-like files, but not components.Component is a reusable function that returns the element that you can render on the page. Utils are the helpers functions that you can use, for example, to modify data. So, if you create a "Modal" function that returns div element - it will be a component. But createAddTaskWindow function that returns void is not a component